{"uuid":"2dcb9c08-1317-4c2a-abaa-c9ad75e907b2","name":"Default test","children":["eecd6b3d-e71a-498c-baba-1ea5229f2f02","9bfc8d1f-f915-4e82-9ef9-f84a976d2567","8ea390b3-3bd4-47dd-a441-b97477cce28d","32578fc4-a83b-4989-9448-f064fd51ff56","f5b9a267-cbdb-4b98-9e12-d90e15668074","b39989aa-f9de-488c-b547-e2f02ad4cff2","4fb8a99c-3ad0-4927-94e8-7b636f1c6ee0","620300e1-4d1a-4043-8dcf-bd039b47186e"],"befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver","trace":"org.openqa.selenium.remote.NoSuchDriverException: chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:13)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:99)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:51)\r\n\tat runner.FeatureRunner.setUp(FeatureRunner.java:21)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:652)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699620452833,"stop":1699620453028}],"afters":[{"name":"tearDown","status":"broken","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699620454236,"stop":1699620454237}],"start":1699620452772,"stop":1699620454240}